test/runify.js

```diff
@@ -56,6 +56,72 @@ hosts.forEach(function (record) {
       }
     });
 
+    describe('child process creation', function() {
+      let agent;
+
+      before(function() {
+        return runify.createAgent(type, options)
+          .then(a => agent = a);
+      });
+
+      after(function() {
+        return agent.destroy();
+      });
+
+      it('forwards hostArgs (via agent.args) correctly', function () {
+        let args = [];
+
+        // The arguments in agent.args are those provided
+        // via the --hostArgs options when called from
+        // test262-harness
```

Comment:
How did you decide where to wrap these lines? In absence of any local convention, I'd recommend 80 characters for readability's sake.